go/vt/wrangler/materializer.go

```diff
@@ -63,8 +63,9 @@ type materializer struct {
 }
 
 const (
-	createDDLAsCopy               = "copy"
-	createDDLAsCopyDropConstraint = "copy:drop_constraint"
+	createDDLAsCopy                 = "copy"
+	createDDLAsCopyDropConstraint   = "copy:drop_constraint"
```

Comment:
Can we say authoritatively that we will never need to drop all constraints? If so, we will need to run a deprecation cycle to remove `copy:drop_constraint`, since it is possible there are applications using this flag at the moment.

Reply:
@Phanatic Why wasn't it replaced endtoend instead of only at the user interface? I would have expected the property here to be renamed instead of a new one being added.

Reply:
Yeah, that makes sense.